docker/tools-docker-tests.test.ts

```diff
@@ -0,0 +1,151 @@
+import {expect} from "chai";
+import {ethers} from "hardhat";
+import fs from "fs";
+import path from "path";
+import shelljs from "shelljs";
+import {ConsensusContracts} from "../src/pessimistic-utils";
+
+const deployOutput = JSON.parse(fs.readFileSync(path.join(__dirname, "./deploymentOutput/deploy_output.json"), "utf8"));
+const {polygonRollupManagerAddress, polygonZkEVMBridgeAddress, polygonZkEVMGlobalExitRootAddress, polTokenAddress} =
+    deployOutput;
+const createRollupOutput = JSON.parse(
+    fs.readFileSync(path.join(__dirname, "./deploymentOutput/create_rollup_output.json"), "utf8")
+);
+const {rollupAddress} = createRollupOutput;
+import {
+    PolygonRollupManager,
+    PolygonPessimisticConsensus,
+    PolygonValidiumEtrog,
+    PolygonZkEVMEtrog,
+} from "../typechain-types";
+
+describe("Tooling docker build tests Contract", () => {
+    it("Create a new rollup", async () => {
+        // Read docker deployment output
+        const dockerCreateRollupOutput = JSON.parse(
+            fs.readFileSync(path.join(__dirname, "./deploymentOutput/create_rollup_output.json"), "utf8")
+        );
+        const dockerDeploymentOutput = JSON.parse(
+            fs.readFileSync(path.join(__dirname, "./deploymentOutput/deploy_output.json"), "utf8")
+        );
+        // Read create rollup config file
+        const createRollupConfig = JSON.parse(
+            fs.readFileSync(path.join(__dirname, "../tools/createNewRollup/create_new_rollup.json.example"), "utf8")
+        );
+
+        // Update example config from docker deployment output
+        createRollupConfig.consensusContractName = dockerCreateRollupOutput.consensusContract;
+        createRollupConfig.gasTokenAddress = dockerCreateRollupOutput.gasTokenAddress;
+        createRollupConfig.outputPath = "create_new_rollup_output.json";
+        createRollupConfig.chainID = 12;
+        createRollupConfig.rollupManagerAddress = dockerDeploymentOutput.polygonRollupManagerAddress;
+        fs.writeFileSync(
+            path.join(__dirname, "../tools/createNewRollup/create_new_rollup.json"),
+            JSON.stringify(createRollupConfig, null, 2)
+        );
+
+        // Run tool
+        shelljs.exec("npx hardhat run ./tools/createNewRollup/createNewRollup.ts --network localhost");
```

Comment:
take care executing scripts inside scripts ^^, this might eb executed in background to check transpilation errors, and coudl create a rollup D: